include/wally_elements.h

```diff
@@ -162,6 +210,46 @@ WALLY_CORE_API int wally_asset_rangeproof(
     size_t len,
     size_t *written);
 
+/**
+ * Generate an explicit value rangeproof for a transaction output.
```

Comment:
I'd remove the reference to "transaction output", an explicit rangeproof might be useful in more generic oontext.

I'd also add what this actually proves, i.e. that a value commits to a value commitment, without the need to reveal the value blinding factor to the verifier.

Reply:
Removed output ref.

I don't think the wally docs are the place for further documentation, in line with the other function docs it is expected that the caller understands what/wht they need these primitives.